controller/src/main/java/org/jboss/as/controller/OperationContext.java

```diff
@@ -482,6 +482,16 @@
     ServiceTarget getServiceTarget() throws UnsupportedOperationException;
 
     /**
+     * Get the service target.  If the step is not a runtime operation handler step, an exception will be thrown.  The
+     * returned service target is limited such that only the service add methods are supported.  If a service added
+     * to this target was removed by a prior operation step, the install will wait until the removal completes.
+     *
+     * @return the service target
+     * @throws UnsupportedOperationException if the calling step is not a runtime operation step
+     */
+    CapabilityServiceTarget getCapabilityServiceTarget() throws UnsupportedOperationException;
```

Comment:
Is a new method needed here vs just changing the return type of the existing method?